recipes/level-zero/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set version = "1.9.9" %}
+{% set test_version = "1.9.0" %}
```

Comment:
@kurapov-peter If the `test_version` is always guaranteed to be the first two digits of `version` and a zero, perhaps we can be uses Jinja here:

```
{% set test_version = ".".join(version.split('.')[:2] + ['0']) %}
```

This way `'test_version'` will always stay in sync with the `'version'`.

Reply:
Sure, updated the meta.yaml